code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -250,15 +250,15 @@
 		switch(drunkenness)
 			if(11 to 21)
 				msg += "[t_He] [t_is] slightly flushed.\n"
-			if(22 to 41)
+			if(21.01 to 41) //.01s are used in case drunkenness ends up to be a small decimal

```

Comment:
I thought that would make the two categories conflict. If it doesn't, I'll fix it.